### PR TITLE
As far as I understand it's the loadbalancer should forward connectio…

### PR DESCRIPTION
**Pull request by adyatlov**
…ns, not the master.

## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by joel-hamill** [comment]
ping @lingmann for review

**Comment by lingmann** [comment]
🚢

File: 1.7/administration/installing/custom/configuration-parameters.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ This option specifies that Mesos agents are used to discover the masters by givi
...
 
-    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The master should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
+    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The load balancer should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
```
Comment (joel-hamill) [inline 1.7/administration/installing/custom/configuration-parameters.md]: @adyatlov please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.8/administration/installing/custom/configuration-parameters.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ This option specifies that Mesos agents are used to discover the masters by givi
...
 
-    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The master should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
+    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The load balancer should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
```
Comment (joel-hamill) [inline 1.8/administration/installing/custom/configuration-parameters.md]: @adyatlov please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

File: 1.9/administration/installing/custom/configuration-parameters.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ This option specifies that Mesos agents are used to discover the masters by givi
...
 
-    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The master should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
+    *   **exhibitor_address** This required parameter specifies the location (preferably an IP address) of the load balancer in front of the masters. The load balancer must accept traffic on ports 8080, 5050, 80, and 443; and forward it to the same ports on the master (for example, 8080 on lb -> 8080 on one master, 5050 on lb -> 5050 on one master). The load balancer should forward any new connections via round robin, and should avoid machines that do not respond to requests on port 5050 to ensure the master is up.
```
Comment (joel-hamill) [inline 1.9/administration/installing/custom/configuration-parameters.md]: @adyatlov please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.

**Pull request merged**